drivers/serial/Kconfig.pl011

```diff
@@ -3,6 +3,8 @@
 
 menuconfig UART_PL011
 	bool "ARM PL011 UART Driver"
+	default y
```

Comment:
In this file UART_PL011_SBSA likely should be enabled in case DT_HAS_ARM_SBSA_UART_ENABLED